channels/serializers/invites.py

```diff
@@ -0,0 +1,28 @@
+"""Channel invitation serializers"""
+from django.db import transaction
+
+from rest_framework import serializers
+
+from channels.models import ChannelInvitation
+from channels import tasks
+
+
+class ChannelInvitationSerializer(serializers.ModelSerializer):
+    """Serializer for channel invitations"""
+
+    def create(self, validated_data):
+        with transaction.atomic():
```

Comment:
It looks like this code is uncovered by tests. Can you add one or more tests to cover this?